package.json

```diff
@@ -9,8 +9,8 @@
   "license": "MIT",
   "author": "devsupport@webex.com",
   "engines": {
-    "node": "^10.x",
-    "npm": "^6.x"
+    "node": "^14.x",
+    "npm": "6.x"
```

Comment:
do we still need engines now that we can use current LTS?

Reply:
> Unless the user has set the engine-strict config flag, this field is advisory only and will only produce warnings when your package is installed as a dependency.

It is helpful to have so let's keep them in.